### DISCUSSION TITLE
I cannot Use VIdeo

### DISCUSSION THREAD

**Discussion by Keyaanm1** [aryasoni98/AI-Interview#109] [Q&A]
The project is super cool! I, however, cannot use the video feature as it says cannot POST /video_Interview

**Comment by hubblehox** [comment]
Hi Arya,

We're building an AI Interview module
I saw your github repo,
Can we have a discussion ?

-- 
Best,
Souvik Roy
Mob : +91-7980991392
Skype : souvikroy1
